src/shared/credentials/accountId.ts

```diff
@@ -26,3 +27,27 @@ export async function getAccountId(credentials: AWS.Credentials, region: string)
         return undefined
     }
 }
+
+/**
+ * Temporary workaround for missing credentials bug.
+ * TODO: Remove this function when migrated over to V3 of the SDK
+ */
+export async function getAccountIdHack(credentials: AWS.Credentials, region: string): Promise<string | undefined> {
+    // In V2 of the AWS JS SDK, the STS client tries to parse from the credentials file even
+    // if it doesn't exist, throwing an uncaught exception.
+    // This is because it doesn't check for the region we passed to it...
+    // Instead we have to set an enviromental variable
+    const env = process.env as EnvironmentVariables
+    const tmp = env.AWS_REGION
+    env.AWS_REGION = env.AWS_REGION ?? region
```

Comment:
if `env.AWS_REGION` is empty string (as opposed to undefined), then empty string will be used. Is that intended?

Reply:
The SDK appears to treat an empty string as undefined, so we need to explicitly check for it in our code.